### DISCUSSION TITLE
Partition in reverse order

### DISCUSSION THREAD

**Discussion by j-hap** [gephi/gephi#2653] [Q&A]
Hi, I hope this feature exists and I'm just blind: When coloring nodes by partition (let's say in-degree), the list of the categories is always sorted ascending. meaning that the low in-degree are listed on top. When using a palette with a limited number of colors, those colors are assigned to the top entries in the list. On top, there are the classes with low in-degree. Nodes with a low in-degree are most likely not the ones you want to highlight. Is there any chance to either
- reverse the order of the list?
- put the non-default palette colors at the bottom of the list?

thanks!

**Comment by mbastian** [comment]
Hi @j-hap, the list is sorted descending by frequency. Sorting by the value itself I don't really see the use-case but certainly inverting the sort so that the lowest frequency are first could make sense in certain use-cases. Would that make sense?

**Comment by j-hap** [comment]
Ah I see, so it's just coincidence that the order matches the degree in my case. An ascending sort by frequency would be very helpful. I do also see sense in a sort-by-value though. If one wanted to color only the nodes with a high or low in-degree, regardless of how frequent these degrees occur, would be helpful as well.
